### PR TITLE
Add a BasicDateInput in ScenarioParameters

### PR DESCRIPTION
**Pull request by COSMO-VMI**

### CODE REVIEW

**Review by csm-thu — APPROVED** [review]
LGTM

File: src/components/ScenarioParameters/components/BasicTypes.js [hunk 1]
```diff
@@ -61,6 +63,12 @@ const BasicTypes = ({
     id: 'standard-required'
   };
 
+  const dateProps = {
+    disabled: !editMode,
+    id: 'standard-required',
```
Comment (jreynard-code) [inline src/components/ScenarioParameters/components/BasicTypes.js:68]: Could you please change the id and ids above, they sould be unique?

File: src/components/ScenarioParameters/components/BasicTypes.js [hunk 2]
```diff
@@ -89,6 +97,12 @@ const BasicTypes = ({
           changeSwitchType={changeSwitchType}
           switchProps={switchFieldProps}
         />
+        <BasicDateInput
+          classes={classes}
+          label='Pick a date'
```
Comment (jreynard-code) [inline src/components/ScenarioParameters/components/BasicTypes.js:102]: Could you please use 'useTranslation' hook here and above to change label regarding language?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM, just little changes

**Pull request merged**